packages/headless/src/api/generated-answer/generated-answer-client.ts

```diff
@@ -104,6 +104,7 @@ export class GeneratedAnswerAPIClient {
     const stream = () =>
       fetchEventSource(buildStreamingUrl(url, organizationId, streamId), {
         method: 'GET',
+        fetch,
```

Comment:
What's that parameter being added does exactly?
Assuming this is us injecting what "fetch" client to use for the event source library?

Reply:
Exactly yeah!

if we don't pass this parmeter, by default it uses `window.fetch`, however the ponyfill we are using is exporting `fetch` so it will not take any effect when you can `window.fetch`

So I'm passing this parameter here to make the library use `fetch` aka our ponyfill aboratable fetch.